source/core/src/main/com/csse3200/game/components/player/PlayerInventoryDisplay.java

```diff
@@ -9,10 +8,14 @@
 import com.badlogic.gdx.scenes.scene2d.InputListener;
 import com.badlogic.gdx.scenes.scene2d.ui.*;
 import com.badlogic.gdx.scenes.scene2d.utils.ChangeListener;
-
+import com.csse3200.game.ai.tasks.AITaskComponent;
+import com.csse3200.game.components.tasks.ItemProximityTask;
+import com.csse3200.game.components.tasks.UseItemTask;
 import com.badlogic.gdx.utils.Align;
```

Comment:
Change this file to only have relevant changes to the item usage (ie no need to modify any formatting etc. with hotbar)

Reply:
Check what the current version of this file looks like in the main team 3 branch